cmd/skupper/skupper.go

```diff
@@ -862,14 +884,28 @@ func init() {
 
 	cmdCompletion := NewCmdCompletion()
 
-	rootCmd = &cobra.Command{Use: "skupper"}
+	rootCmd = &cobra.Command{
+		Use:                    "skupper",
+		BashCompletionFunction: bash_completion_func,
```

Comment:
@nicob87 I would suggest writing the bash_completion in a separate .sh file,
which would make it way easier to maintain as we go.

Then we could tweak the build command for the `skupper` binary to include a
build-time variable that points to the `bash_completion.sh` file, like:

```
go build -ldflags "-X main.BashCompletionScript=`readlink -f bash_completion.sh`" ...
```

Reply:
I tested it locally using a 65k shell script and it worked just fine.

Reply:
mmmm and that works for windows and macos? (windows does not have  a cat, to begin) will investigate options anyway...
awesome review!

Reply:
maybe just the "bash_completion_func" variable definition to a dedicated *.go file is a simple solution, if you want that file to be a ".sh" I think we should "read" that file from our go code, but, this makes a dependence and a possible runtime error, on a .sh file, which I do not like so much ...

Reply:
The dependency exists only at build-time, after that it no longer exists, but a separate go file just with the constant
might also work.